benchmark/venv.py

```diff
@@ -193,6 +206,25 @@ def create_virtualenv():
     return venv_python
 
 
+def create_virtualenv_py3(venv, venv_path):
+    class BenchmarkEnv(venv.EnvBuilder):
```

Comment:
The API isn't nearly as nice as the one in py3's stdlib: https://github.com/pypa/virtualenv/blob/master/virtualenv.py

I can look into it and see if its viable today.